### PR TITLE
chore: update slider label example to use range min and max settings

### PR DESCRIPTION
**Pull request by bheston**
# Description

The lower and upper slider labels should use the special min and max positioning values in the example.

## Motivation & context

Alignment is not correct and doesn't illustrate the preferred usage pattern for proper alignment.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-component-explorer/app/utilities/configs/slider.ts [hunk 1]
```diff
@@ -36,36 +37,36 @@ const sliderConfig: ComponentViewConfig<SliderProps> = {
                     {
                         id: sliderLabelSchema.id,
                         props: {
-                            valuePositionBinding: 0,
-                            label: "low",
+                            valuePositionBinding: SliderTrackItemAnchor.totalRangeMin,
```
Comment (janechu) [inline sites/fast-component-explorer/app/utilities/configs/slider.ts:40]: There is a large effort to update this entire app on a feature branch. If this is not urgent can we hold off?

Comment (nicholasrice) [reply sites/fast-component-explorer/app/utilities/configs/slider.ts:40]: which pr / issue / branch is this pending on?

Comment (bheston) [reply sites/fast-component-explorer/app/utilities/configs/slider.ts:40]: Jane said `feature/update-explorer-site-to-tooling-v2-and-web-components`. I haven't looked into making the change there as well yet.

Comment (janechu) [reply sites/fast-component-explorer/app/utilities/configs/slider.ts:40]: I forgot to post that here, but a new wrinkle was introduced just now where we're discussing placing only web components in the component explorer. Let's take this discussion to discord and then we can update this PR with the resolution.

**Pull request closed without merging**